anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/forecast/point/PewmaPointForecaster.java

```diff
@@ -129,17 +106,36 @@ private void updateEstimates(double value) {
         updateMeanAndStdDev();
     }
 
-    private void updateMeanAndStdDev() {
-        this.mean = this.s1;
-        this.stdDev = Math.sqrt(this.s2 - this.s1 * this.s1);
-    }
-
     private double calculateAlpha(double pt) {
-        val params = getParams();
         if (this.trainingCount < params.getWarmUpPeriod()) {
             this.trainingCount++;
             return 1.0 - 1.0 / this.trainingCount;
         }
         return (1.0 - params.getBeta() * pt) * this.adjAlpha;
     }
+
+    @Data
+    @Accessors(chain = true)
+    public static class Params {
+
+        /**
+         * Smoothing param.
```

Comment:
Let's add a TODO for "Describe why we chose 0.15, 1.0, 0.0, 30 as defaults for this class."

Reply:
We probably want to be a little selective here. If the paper recommends certain defaults then I think that's worth documenting. In the case of the initial mean estimate, it was just that 0.0 is a less-strange default than 4.3 would have been. I'll add the TODO though.